src/components/ContactForm/index.test.js

```diff
@@ -73,11 +67,6 @@ describe("LeaveMessageForm", () => {
     fireEvent.submit(screen.getByRole("button"));
 
     await waitFor(() => expect(screen.queryAllByRole("alert")).toHaveLength(0));
-    expect(mockSubmit).toBeCalledWith({
```

Comment:
why did this get deleted

Reply:
usually when someone deletes a test it means they're breaking something (watch that joma tech video about code reviews at big companies)

Reply:
me and Jamil are trying to fix this issue, because after I added translation, test started failing. maybe @Jamil-Bailony you can explain why you deleted?

Reply:
after I pulled and pushed new code, this line is back as far as I understand. one test is still failing, I am not sure what's going on

Reply:
so whats the error?

Reply:
<img width="1440" alt="Screen Shot 2020-11-18 at 15 00 34" src="https://user-images.githubusercontent.com/67760504/99528142-e51f1580-29ae-11eb-8152-f3f2fcdedfec.png">

Reply:
I delete that piece of code which is a mocking function of submit parameter sent to the component parent. As we implemented message sending functionality inside the component itself, we did not need that mock function anymore.

Also, we had some issues related to using i18next with jest and react testing library as Jennet mentioned and we were in a jam yesterday. However, we found a way to get around it using aria-label to catch the elements.

Reply:
I fixed the test but I have this issue now.
<img width="1062" alt="Screen Shot 2020-11-18 at 13 44 05" src="https://user-images.githubusercontent.com/67760504/99539214-012ab300-29bf-11eb-8e18-9d01dd9c0721.png">

I tried to run each test function separately and found out that this function is causing this issue 

` await fireEvent.submit(screen.getByRole("button"));
    await waitFor(() => expect(screen.queryAllByRole("alert")).toHaveLength(0));
    expect(screen.getByRole("textbox", { name: /Full Name/i }).value).toBe("");
    expect(screen.getByRole("textbox", { name: /Email address/i }).value).toBe(
      ""
    );
    expect(screen.getByRole("textbox", { name: /message/i }).value).toBe("");`

Reply:
okay, after 3 hours of trying to fix it, it works. can you please check @louisrli ?